modules/esp8266-wifi/pages/index.adoc

```diff
@@ -0,0 +1,30 @@
+= WiFi end-to-end
+
+This workshop focuses on the use case of reporting data from an embedded device, to the cloud, and receive commands
+back.
+
+The idea is that you have some embedded device, with a WiFi modem, periodically reporting data to the cloud, and
+occasionally receiving back commands to control the device.
+
+== Pre-requisites
+
+This workshop requires additional hardware, as we need an embedded device with a WiFi modem.
+
+Additionally to the common pre-requisites, you will need:
+
+* A Drogue Device supported development board and WiFi modem
+* A WiFi network
+* A publicly accessible Drogue IoT Cloud instance running version 0.5+
```

Comment:
Maybe point to the sandbox if people want to focus on the embedded side.

Reply:
Agreed, we've got the drogue cloud setup covered already elsewhere I guess.